src/SmartComponents/SystemDetail/InventoryDetail.js

```diff
@@ -11,14 +10,17 @@ import { setPageTitle, useFeatureFlag } from '../../Utilities/Hooks';
 import { InventoryDetailHead, AppInfo, DetailWrapper } from '@redhat-cloud-services/frontend-components/Inventory';
 import { Alert, Grid, GridItem, TextContent, Text } from '@patternfly/react-core';
 import { fetchSystemDetailsAction } from '../../store/Actions/Actions';
-import propTypes from 'prop-types';
 import { clearNotifications } from '@redhat-cloud-services/frontend-components-notifications/redux';;
 import ErrorHandler from '../../PresentationalComponents/Snippets/ErrorHandler';
 import PatchSetWrapper from '../../PresentationalComponents/PatchSetWrapper/PatchSetWrapper';
 import usePatchSetState from '../../Utilities/usePatchSetState';
 import { featureFlags } from '../../Utilities/constants';
+import { paths } from '../../Routes';
+import { useParams } from 'react-router-dom';
+
+const InventoryDetail = () => {
+    const {  inventoryId } = useParams();
```

Comment:
I'm not sure why yet, but this triggers a 🚩in me.

Reply:
Why is that? The route has` :inventoryId `inside of it and this just grabs that arg from the url. 
In the scenarios I can think of where the url can be messed up, it would break inventory in the old version as well